content/enterprise_influxdb/v1.9/tools/influxd-ctl.md

```diff
@@ -246,6 +246,10 @@ Optional arguments are in brackets.
 
 Name of the single database to back up.
 
+###### [ `-estimate` ]
+
+Provide estimated backup size and progress messages during backup.
```

Comment:
Do we want an example output?

Reply:
If you have one handy, that'd be great!

Reply:
> Backing up node backup_data_0_1:8088, db stress, rp autogen, shard 14
Files: 8 / 9	Bytes: 189543424 / 231921095	Completed: 82% in 22s	Estimated remaining: 3s
Files: 8 / 9	Bytes: 189543424 / 231921095	Completed: 82% in 23s	Estimated remaining: 2s
Files: 9 / 9	Bytes: 231736320 / 231921095	Completed: 100% in 24s	Estimated remaining: 447µs
Done backing up node backup_data_0_1:8088, db stress, rp autogen, shard 14 in 67ms: 42192896 bytes transferred

From:
https://github.com/influxdata/feature-requests/issues/61#issuecomment-932632697